.changeset/happy-falcons-taste.md

```diff
@@ -0,0 +1,5 @@
+---
+'@roboplay/robo.js': patch
+---
+
+Patch(robo): add FlashcoreOptions to has method.
```

Comment:
There's no need to include the package scope in changesets btw. That only applies to commit titles. It's better to use the feature as the scope. :3